telemetry/glean/relay-server-metrics.yaml

```diff
@@ -0,0 +1,36 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# For more information on configuring this file:
+# https://mozilla.github.io/glean/book/reference/yaml/metrics.html
+
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+email:
+  generate_mask:
+    type: event
+    description: An email mask is generated.
+    expires: never
+    bugs:
+      - http://example.com
+    data_reviews:
+      - http://example.com
```

Comment:
question (blocking): does these need real values before merge?

Reply:
I'm coming around to not merging this PR - maybe this is just a spike.